tests/main.tf

```diff
@@ -797,7 +808,7 @@ module "vpn-gateway" {
 module "web-alb" {
   source = "../modules/web-alb"
 
-  dns_name               = ""
+  dns_name               = "test-web-alb.net"
```

Comment:
The params should be empty, `make plan` won't pass. This test is just aiming to validate module by initializing them in the simplest way possible. We probably want to keep this empty.

Reply:
Ok, I'll revert this. I practice `web-alb` fails because of missing TLS certificate for this name.

Reply:
Done